packages/core/lib/patchers/http_p.js

```diff
@@ -138,7 +139,11 @@ function enableCapture(module, downstreamXRayEnabled, subsegmentCallback) {
         subsegment.close(e);
       }
 
-      if (this._events && this._events.error && this._events.error.length === 1) {
+      // Only need to remove our listener & re-emit if we're not listening using the errorMonitor,
+      // e.g. the app is running on Node 10. Otherwise the errorMonitor will re-emit automatically.
+      // See: https://github.com/aws/aws-xray-sdk-node/issues/318
+      // TODO: Remove this logic once node 12 support is deprecated
+      if (!events.errorMonitor && this.listenerCount('error') <= 1) {
         this.removeListener('error', errorCapturer);
```

Comment:
What happens when `this.listenerCount('error')` return 0 (which semantically means there is no error listener) and we call `this.removeListener('error', errorCapturer);`? Would there be an error?